### PR TITLE
Minor English/style/formatting changes of the man pages.

### PR DESCRIPTION
**Pull request by fnevgeny**

### CODE REVIEW

File: man/pdfpcrc.in [hunk 1]
```diff
@@ -65,11 +65,11 @@ Percentage of the presenter screen to be used for the current slide. (int, Defau
 .TP
 .BI disable-caching
 see
-.B pdfpcrc(5)
+.B pdfpc(1)
```
Comment (AndreasBilke) [inline man/pdfpcrc.in]: The information about `disable-caching` is in pdfpcrc(5)

File: man/pdfpcrc.in [hunk 2]
```diff
@@ -65,11 +65,11 @@ Percentage of the presenter screen to be used for the current slide. (int, Defau
...
 .TP
 .BI disable-compression
 see
-.B pdfpcrc(5)
+.B pdfpc(1)
```
Comment (AndreasBilke) [inline man/pdfpcrc.in]: Same here

File: man/pdfpcrc.in [hunk 3]
```diff
@@ -82,7 +82,7 @@ Switch the presentation and the presenter screen. (bool, Default false)
 .TP
 .B time-of-day
 see
-.B pdfpcrc(5)
+.B pdfpc(1)
```
Comment (AndreasBilke) [inline man/pdfpcrc.in]: Same here

File: man/pdfpcrc.in (hunk 1 above)
Comment (fnevgeny) [reply man/pdfpcrc.in]: Sorry, I am probably confused. Where is it in pdfpcrc(5)? I rather see it described in pdfpc(1), section "Caching / Pre-rendering" under "FEATURES". In any case, self-referring pdfpcrc(5) from within pdfpcrc(5) is strange.

Comment (AndreasBilke) [reply man/pdfpcrc.in]: Oh sorry! You are right and I am were confused. I was thinking you changed this in pdfpc(1)... So, yes the reference to pdfpc(1) is correct here!

**Pull request merged**